### PR TITLE
Ch17 fixes

### PR DESCRIPTION
**Pull request by JDuchniewicz**

### CODE REVIEW

File: 17_foundations.ipynb [hunk 1]
```diff
@@ -1066,7 +1066,7 @@
     "\n",
     "```\n",
     "Image  (3d tensor): 256 x 256 x 3\n",
-    "Scale  (1d tensor):  (1)   (1)  3\n",
+    "Scale  (2d tensor):  (1)   (1)  3\n",
```
Comment (jph00) [inline 17_foundations.ipynb:1069]: Why did you change this? It appears to be a 1d tensor AFAICT?

**Review by jph00 — CHANGES_REQUESTED** [review]
Thanks so much for the fixes! Please fix the '1d' issue (or let me know if I made a mistake) and then I can merge

File: 17_foundations.ipynb (hunk 1 above)
Comment (JDuchniewicz) [reply 17_foundations.ipynb:1069]: I am not sure if this is consistent with the description above:
> Arrays do not need to have the same number of dimensions. For example, if you have a 256×256×3 array of RGB values, and you want to scale each color in the image by a different value, you can multiply the image by a **one-dimensional array with three values.**

Maybe I am mistaken, but this caught my attention.

Comment (jph00) [reply 17_foundations.ipynb:1069]: It is a one-dimensional tensor, as it says, so it should say "1d" in the summary, not "2d".

Comment (JDuchniewicz) [reply 17_foundations.ipynb]: Right, I fixed the wrong tensor and this confused me. Sorry.

File: 17_foundations.ipynb [hunk 2]
```diff
@@ -1066,15 +1066,15 @@
...
-    "Scale  (2d tensor):  (1)   (1)  3\n",
+    "Scale  (1d tensor):  (1)   (1)  3\n",
     "Result (3d tensor): 256 x 256 x 3\n",
     "```\n",
     "    \n",
     "However, a 2D tensor of size 256×256 isn't compatible with our image:\n",
     "\n",
     "```\n",
     "Image  (3d tensor): 256 x 256 x   3\n",
-    "Scale  (1d tensor):  (1)  256 x 256\n",
```
Comment (JDuchniewicz) [inline 17_foundations.ipynb:1077]: This is the place with the proper fix

Comment (jph00) [reply 17_foundations.ipynb:1077]: Yes - please go ahead and fix that one! :)

Comment (jph00) [reply 17_foundations.ipynb:1077]: Ah I see you already did :)
